src/frontend/js/widgets/Dashboard/components/DashboardItem/Order/DashboardItemOrder.tsx

```diff
@@ -26,6 +27,27 @@ const messages = {
     description: 'Accessible label displayed while certificate is being fetched on the dashboard.',
     defaultMessage: 'Loading certificate...',
   },
+  contractUnsigned: {
+    id: 'components.DashboardItemOrderContract.contractUnsigned',
```

Comment:
To keep id consistent respect the naming convention `components.<COMPONENT_NAME>.messageKey`

```suggestion
    id: 'components.DashboardItemOrder.contractUnsigned',
```